.github/workflows/test.yml

```diff
@@ -40,6 +40,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements-dev.txt
+          pip install pyTRLCConverter>=2.0.0
```

Comment:
Don't you need to quote the argument ? 

I wouldn't be surprised if this installes whatever pyTRLCConverter and redirect the pip output into a file called `=2.0.0`